### PR TITLE
Fix documentation of firewall rule exclude source/dest

### PR DESCRIPTION
**Pull request by asarfaty**

### CODE REVIEW

File: website/docs/r/lb_pool.html.markdown [hunk 1]
```diff
@@ -44,6 +51,41 @@ resource "nsxt_lb_pool" "lb_pool" {
...
+  algorithm                = "LEAST_CONNECTION"
+  min_active_members       = 1
+  tcp_multiplexing_enabled = false
+  tcp_multiplexing_number  = 3
+  active_monitor_id        = "${nsxt_lb_icmp_monitor.lb_icmp_monitor.id}"
+  passive_monitor_id       = "${nsxt_lb_passive_monitor.lb_passive_monitor.id}"
+  snat_translation_type    = "LbSnatAutoMap"
+ 
+  member_group {
+<<<<<<< HEAD
```
Comment (annakhm) [inline website/docs/r/lb_pool.html.markdown]: some balagan here..

File: nsxt/resource_nsxt_lb_pool.go [hunk 2]
```diff
@@ -221,6 +294,7 @@ func resourceNsxtLbPoolCreate(d *schema.ResourceData, m interface{}) error {
 		TcpMultiplexingEnabled: tcpMultiplexingEnabled,
 		TcpMultiplexingNumber:  tcpMultiplexingNumber,
 		Members:                members,
+		MemberGroup:            memberGroup,
```
Comment (annakhm) [inline nsxt/resource_nsxt_lb_pool.go]: does the backend behave well when both members and member groups are specified?

File: website/docs/r/firewall_section.html.markdown [hunk 3]
```diff
@@ -85,7 +87,7 @@ The following arguments are supported:
   * `action` - (Required) Action enforced on the packets which matches the firewall rule. [Allowed values: "ALLOW", "DROP", "REJECT"]
   * `applied_to` - (Optional) List of object where rule will be enforced. The section level field overrides this one. Null will be treated as any. [Supported target types: "LogicalPort", "LogicalSwitch", "NSGroup"]
   * `destination` - (Optional) List of the destinations. Null will be treated as any. [Allowed target types: "IPSet", "LogicalPort", "LogicalSwitch", "NSGroup", "MACSet" (depending on the section type)]
-  * `destinations_excluded` - (Optional) Negation of the destination.
+  * `destinations_excluded` - (Optional) When this boolean flag is set to true, the rule destinations will be negated.
   * `direction` - (Optional) Rule direction in case of stateless firewall rules. This will only considered if section level parameter is set to stateless. Default to IN_OUT if not specified. [Allowed values: "IN", "OUT", "IN_OUT"]
   * `disabled` - (Optional) Flag to disable rule. Disabled will only be persisted but never provisioned/realized.
```
Comment (annakhm) [inline website/docs/r/firewall_section.html.markdown]: just noticed - its really weird to name this "disabled" and not "enabled".. guess its too late now

Comment (asarfaty) [reply website/docs/r/firewall_section.html.markdown]: You are right, but it will be a bad idea to change this now.

**Comment by asarfaty** [comment]
This patch is a mess due to unintended merge. I'll post a new one

**Pull request closed without merging**